### PR TITLE
tls updates

### PR DESCRIPTION
**Pull request by tanner0101**
Moves to CTLS

### CODE REVIEW

File: Sources/Vapor/Cipher/CryptoCipher.swift [hunk 1]
```diff
@@ -37,8 +37,6 @@ extension CryptoCipher: ConfigInitializable {
 
         let method: Cipher.Method
         switch methodString {
-        case "chacha20":
```
Comment (LoganWright) [inline Sources/Vapor/Cipher/CryptoCipher.swift]: is there a warning somewhere else for users who might have this?

Comment (tanner0101) [inline Sources/Vapor/Cipher/CryptoCipher.swift]: ```throw Error.config("Unknown cipher method '\(methodString)'.")```

Comment (tanner0101) [inline Sources/Vapor/Cipher/CryptoCipher.swift]: i added 
```swift
            if methodString == "chacha20" {
                print("Warning: chacha20 cipher is no longer available. Please use aes256 instead.")
            }
            throw Error.config("Unknown cipher method '\(methodString)'.")
```

Comment (vzsg) [inline Sources/Vapor/Cipher/CryptoCipher.swift]: The basic-template will have to be updated for 2.0, as chacha20 is actually [the default](https://github.com/vapor/basic-template/blob/master/Config/crypto.json) in its crypto config.

**Pull request merged**